packages/cli/lib/base-generator.js

```diff
@@ -21,11 +22,113 @@ module.exports = class BaseGenerator extends Generator {
    * Subclasses can extend _setupGenerator() to set up the generator
    */
   _setupGenerator() {
+    this.option('json', {
+      type: String,
+      description: 'Stringified JSON object to configure options',
```

Comment:
It makes me wonder how practical it is to pass JSON strings via exec arguments. IIRC, there are limitations on the maximum supported length - e.g. Linux limits the length to ~100KB according to [this post](https://unix.stackexchange.com/questions/120642/what-defines-the-maximum-size-for-a-command-single-argument), Windows has a limit of 32KB according to [this post](https://blogs.msdn.microsoft.com/oldnewthing/20031210-00/?p=41553/).

In my experience, CLI programs usually accept large content from stdin stream. 

I am proposing to slightly different set of CLI options and behavior:

```bash
# Read JSON instructions from the given file
$ lb4 app --json commands.json
# No file was provided, read instruction from stdin
$ cat commands.json | lb4 app --json
```

Thoughts?